src/components/BlockCard.vue

```diff
@@ -0,0 +1,204 @@
+      actionNum,
+      nextBlock,
+      previousBlock,
+      numberWithCommas,
+      formatDate,
+      copy,
+      transactions
+    };
+  }
+});
+</script>
+
+<template lang="pug">
+.row.full-width.justify-center
+  .col-xs-12.col-md-8.col-lg-6
+    q-card(flat class="transaction-card")
+      .q-pa-md-md.q-pa-sm-sm.q-pa-xs-xs.q-pa-xl-lg
+        q-card-section.q-pl-md
+          .row.q-col-gutter-sm.justify-between
+            .col-auto(class="text-h4 text-bold") Block
+            .col-auto
+              .row.q-col-gutter-sm
+                .col-auto
+                  q-btn.button-primary( @click="previousBlock" flat dense size='md' icon='arrow_back' )
+                .col-auto 
+                  q-btn.button-primary( @click="nextBlock" flat dense size='md' icon='arrow_forward' )
+        
+        q-card-section.q-pt-none
+          .row.items-center
+            .col-11.text-bold.ellipsis {{numberWithCommas(block_num)}}
+            .col-1
+              q-btn.float-right( @click="copy(block_num.toString())" flat round color="black" icon="content_copy" size='sm')
+
+        q-card-section
+          .text-grey-7 SUMMARY
+        q-separator(inset).card-separator
+        q-card-section
+          .row
+            .col-xs-12.col-sm-6
+              .text-body1.text-weight-medium.text-uppercase Producer
+            .col-xs-12.col-sm-6.text-right.text-bold {{producer}}
+        q-separator(inset).card-separator
+        q-card-section
+          .row
+            .col-xs-12.col-sm-6
+              .text-body1.text-weight-medium.text-uppercase Block time
+            .col-xs-12.col-sm-6.text-right.text-bold {{formatDate(timestamp)}}
+        q-separator(inset).card-separator
+        q-card-section
+          .row
+            .col-xs-12.col-sm-6
+              .text-body1.text-weight-medium.text-uppercase CPU usage
+            .col-xs-12.col-sm-6.text-right.text-bold {{cpuUsage + ' μs'}}
+        q-separator(inset).card-separator
+        q-card-section
+          .row
+            .col-xs-12.col-sm-6
+              .text-body1.text-weight-medium.text-uppercase Net usage
+            .col-xs-12.col-sm-6.text-right.text-bold {{netUsage + ' Bytes'}}
+        q-separator(inset).card-separator
+        q-card-section
+          .row
+            .col-xs-12.col-sm-6
+              .text-body1.text-weight-medium.text-uppercase Schedule Version
+            .col-xs-12.col-sm-6.text-right.text-bold {{schedule_version}}
+        q-separator(inset).card-separator
+        q-card-section
+          .row
+            .col-xs-12.col-sm-6
+              .text-body1.text-weight-medium.text-uppercase Transactions
+            .col-xs-12.col-sm-6.text-right.text-bold {{transactions}}
+        q-separator(inset).card-separator
+        q-card-section
+          .row
+            .col-xs-12.col-sm-6
+              .text-body1.text-weight-medium.text-uppercase Actions
+            .col-xs-12.col-sm-6.text-right.text-bold {{actionNum}}
```

Comment:
you can set an an array of `[ {label: 'label', value: 'value'} ]` then just `v-for` starting on line 148 since they all have the same format:
```
        q-separator(inset).card-separator
        q-card-section
          .row
            .col-xs-12.col-sm-6
              .text-body1.text-weight-medium.text-uppercase {{ label}}
            .col-xs-12.col-sm-6.text-right.text-bold {{ value }}
```

Reply:
Made this change.